contrib/win32/openssh/OpenSSHBuildHelper.psm1

```diff
@@ -221,121 +188,98 @@ function Start-OpenSSHBootstrap
-                Write-BuildMsg -AsError -ErrorAction Stop -Message "User choose not to restart the machine to apply the changes."
-            }
-        }
-        elseif($errorCode -ne 0)
+    else 
+    {
+        if($NativeHostArch.ToLower().Startswith('arm'))
         {
-            Write-BuildMsg -AsError -ErrorAction Stop -Message "$packageName installation failed with error code $errorCode."
+            #TODO: Install VS2019 or VS2017 build tools
+            Write-BuildMsg -AsError -ErrorAction Stop -Message "The required msbuild 15.0, or greater, is not installed on the machine."
         }
-        $VSBuildToolsPath = Get-Item(Join-Path -Path $env:VS140COMNTOOLS -ChildPath '../../vc')
-    }
-    else
-    {
+
         $VSBuildToolsPath = Get-Item(Join-Path -Path $env:VS140COMNTOOLS -ChildPath '../../vc')
         Write-BuildMsg -AsVerbose -Message 'VC++ 2015 Build Tools already present.'
-    }
-
-    if($NativeHostArch.ToLower().Startswith('arm') -and !$VS2019Path -and !$VS2017Path)
-    {
-        #TODO: Install VS2019 or VS2017 build tools
-        Write-BuildMsg -AsError -ErrorAction Stop -Message "The required msbuild 15.0 is not installed on the machine."
-    }
-
-    if($OneCore -or ($NativeHostArch.ToLower().Startswith('arm')))
-    {
-        $win10sdk = Get-Windows10SDKVersion
-        if($null -eq $win10sdk)
-        {
-            $packageName = "windows-sdk-10.1"
+        
+        if (!$MSBuildPath -or ($null -eq $env:VS140COMNTOOLS)) {
+            Get-Chocolatey
+            $packageName = "vcbuildtools"
             Write-BuildMsg -AsInfo -Message "$packageName not present. Installing $packageName ..."
-            choco install $packageName --version=$Win10SDKVerChoco --force --limitoutput --execution-timeout 120 2>&1 >> $script:BuildLogFile
-            $win10sdk = Get-Windows10SDKVersion
-            if($null -eq $win10sdk)
+            choco install $packageName -ia "/InstallSelectableItems VisualCppBuildTools_ATLMFC_SDK;VisualCppBuildTools_NETFX_SDK" -y --force --limitoutput --execution-timeout 120 2>&1 >> $script:BuildLogFile
+            $errorCode = $LASTEXITCODE
+            if ($errorCode -eq 3010)
             {
-                Write-BuildMsg -AsError -ErrorAction Stop -Message "$packageName installation failed with error code $LASTEXITCODE."
+                Write-Host "The recent package changes indicate a reboot is necessary. please reboot the machine, open a new powershell window and call Start-SSHBuild or Start-OpenSSHBootstrap again." -ForegroundColor Black -BackgroundColor Yellow
+                Do {
+                    $input = Read-Host -Prompt "Reboot the machine? [Yes] Y; [No] N (default is `"Y`")"
+                    if([string]::IsNullOrEmpty($input))
+                    {
+                        $input = 'Y'
+                    }
+                } until ($input -match "^(y(es)?|N(o)?)$")
+                [string]$ret = $Matches[0]
+                if ($ret.ToLower().Startswith('y'))
```

Comment:
PowerShell string compares are always case insensitive by default.  So you can do:
```powershell
if ($ret -eq 'y')
```
or 
```powershell
if ($ret -like 'y*')
```